src/css/styles.css

```diff
@@ -63,7 +87,13 @@ body {
 
 .navbar {
     border-radius: 0;
+    box-shadow: 0px 2px 5px rgba(100, 100, 100, 0.7);
+    flex: 0 0 auto;
+    height: 50px;
```

Comment:
I am not a huge fan of this fixed size because now we get content overflowing out of the header like this on narrow screens:

<img width="461" alt="screen shot 2017-11-23 at 11 50 14 am" src="https://user-images.githubusercontent.com/74178/33187141-908902f4-d044-11e7-88f6-293e852ada05.png">

That said, the whole layout is pretty broken on narrow screens anyway, so this is alright for now. But we are probably going to have to come back and fix it.

Reply:
Yea, as I mentioned, also not a fan. I've been avoiding trying to do too much to the css because I don't want to approach it in a piecemeal fashion.

I'd like to have a design session convo and consider a holistic approach that we could apply to the whole site. Specifically, to answer questions about things like desktop first vs. mobile first, what units we should be using (em, rem, vh, px), accessibility, browser support, etc. We haven't had a chance to do so, but it'd be valuable to get your input and come to agreement to help guide my decision making when coding.

For the time being, I removed the fixed height. Went with the assumption we're doing desktop (full screen) first, and added media queries to increase the margin of the container at breakpoints where the navbar increased in height as the screen width decreased.

Reply:
@lightandluck 
I think a design session convo is a good idea. I can help organize, take notes, and document the outcomes. I'll move the organization of that to slack. Thanks for being forward thinking!

Reply:
:thumbsup:

Reply:
@weatherpattern Cool, thanks for handling the logistics!